libs/components/tiles/src/lib/modules/tiles/tile-dashboard/tile-dashboard.component.scss

```diff
@@ -55,7 +55,7 @@
 }
 
 @include mixins.sky-theme-modern {
-  @include mixins.sky-host-responsive-container-sm-min() {
+  :host-context(.sky-tile-dashboard-gt-xs) {
```

Comment:
Have you tested this in Firefox? I've had instances where these pseudo-class functions don't work in Firefox when inside the theme mixins since those also use `host-context()`.

Reply:
It seems to be working. This component still uses emulated view encapsulation, which might be the difference.